codec_test.go

```diff
@@ -125,7 +125,7 @@ var initializeResponseStruct = InitializeResponse{
 var configurationDoneRequestString = `{"seq":2,"type":"request","command":"configurationDone"}`
 var configurationDoneRequestStruct = ConfigurationDoneRequest{
 	Request:   *newRequest(2, "configurationDone"),
-	Arguments: ConfigurationDoneArguments{},
```

Comment:
This one puzzles me - why did the empty struct here not lead to an `"arguments":{}` addition in line 125, like in line 158/161?